packages/devtools_app/lib/src/screens/vm_developer/vm_developer_tools_screen.dart

```diff
@@ -60,15 +61,22 @@ class VMDeveloperToolsScreen extends Screen {
 }
 
 class VMDeveloperToolsScreenBody extends StatefulWidget {
-  const VMDeveloperToolsScreenBody();
+  const VMDeveloperToolsScreenBody({
+    this.displayObjectInspector = false,
```

Comment:
Instead of making this a parameter, I'd just add a constant at the top of the file:

```dart
/// Set to true to enable the Objects tab.
const displayObjectInspector = false;
```

Then you should be able to make everything `const` further down:

```dart
static const List<VMDeveloperView> views = [
  VMStatisticsView(),
  IsolateStatisticsView(),
  if (displayObjectInspector) ObjectInspectorView(),
];
```